src/core/src/bootstrap/Constants.py

```diff
@@ -257,6 +258,7 @@ class PatchAssessmentSummaryStartedBy(EnumBackport):
 
     # Maintenance Window
     PACKAGE_INSTALL_EXPECTED_MAX_TIME_IN_MINUTES = 5
+    PACKAGE_INSTALL_EXPECTED_AVG_TIME_IN_MINUTES = 2
```

Comment:
Please leave a comment on how this value was picked (or _should_ be picked) so if someone wants to adjust it later (or want reasonable answers to questions like why not 3 or why not 1), they know the constraints to work with. While the MAX time is a design cap, AVG suggests a measured average but is really intended in code to be a 'max' on estimated average.

True average (or our max tolerance for the upper limit of 'average') varies with a number of dynamic variables:
- the package manager
- the batch size (quite importantly this)
- variances in package installations over time
- composition of the fleet managed

Reply:
Added inline comment.